source/code/source/playfab/PlayFabCallRequestContainer.cpp

```diff
@@ -22,8 +22,16 @@ namespace PlayFab
         errorWrapper.UrlPath = url;
 
         Json::Value request;
-        Json::Reader reader;
-        bool parsingSuccessful = reader.parse(requestBody, request);
+        std::string errs;
+        Json::CharReaderBuilder builder;
+        Json::CharReader* reader = builder.newCharReader();
+
+        const char* reqBod = requestBody.c_str();
+        size_t reqBodLength = requestBody.length();
+
+        bool parsingSuccessful = reader->parse(reqBod, reqBod + reqBodLength, &request, &errs);
+
+        delete reader;
```

Comment:
what motivated this change? It doesn't seem to have any relation to avoiding copies or auto

Reply:
This was showing up as a warning on the mac. I can pull it into its own PR.

Reply:
it can be its own PR if you'd like but I don't mind it being here.
I just think miscellaneous changes suckerfished onto other work should be explicitly called out/explained in the description if they are non-obvious. In this case I'd be interested to know what the warning is since there's nothing that stood out to me in the original block that would be substantially different (unless it was just a "deprecated API" warning or something)

Reply:
The warning stated Json::Reader was deprecated. The current jsoncpp github used the above as an example. I do like the unique_ptr change though. I'll be adding that in my next commit first.

Reply:
ah fair enough! then the change you've made is righteous, I just think it's worth calling out in the description so that others can figure out why the change was made later